CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.6.0 (2021-03-05)
+
+- Added a filter fixture. [#220](https://github.com/blackbaud/skyux-lists/pull/220)
```

Comment:
```suggestion
- Added a filter test fixture. [#220](https://github.com/blackbaud/skyux-lists/pull/220)
```
Not sure if `test` is necessary, but that's the language we use in the PR, so I thought I should at least check. Please disregard the suggestion if we don't need it.